docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -0,0 +1,107 @@
+.. _start_quick_start_dynamic_filesystem:
+
+Configuration: Dynamic from filesystem
+======================================
+
+You can start Envoy with dynamic configuration that uses files as a form of
+:ref:`Extensible discovery service (xDS) <xds_protocol>`.
```

Comment:
nit: I think the "x" stands fro "*" effectively. I don't know that we actually have spelled out name for this, but I don't think it's extensible. :)

Reply:
yep, so i searched on this to see if it was ever used previously, and no prior art  that i could find, but...

by convention, that is what a small `x` prefix in an acronym ~is (imhe anyway)

not sure "asteriskds" has the same ring to it 8/

Reply:
i guess this is why i always read it that way...

https://en.wikipedia.org/wiki/XML

Reply:
"x discovery service" so like something you have to pay for in a hotel room

Reply:
to quote lewis carrol

"When I use a word," Humpty Dumpty said, in rather a scornful tone, "it means just what I choose it to mean—neither more nor less." "The question is," said Alice, "whether you can make words mean so many different things." "The question is," said Humpty Dumpty, "which is to be master—that's all."

Reply:
i guess what im saying with my somewhat cryptic posts, is that we should define it (this is more of an issue with the glossary than here)

here, i think its not very helpful for a newcomer to read "...is a form of xds", if xds is not well-defined.

 As i still dont really understand what it means or stands for, i think its not well defined.

Reply:
to be precise, not exactly no prior art https://www.envoyproxy.io/docs/envoy/latest/extending/extending#extending-envoy-for-custom-use-cases

Reply:
so, after going through docs/google a bit more and trying to think how to pin this down...

im still of the opinion that this term is used rather ambiguously, and that to have the level of involvement i now have and to be uncertain exactly what xDS is means something needs clarifying

im coming to the understanding/definition that perhaps xDS *is* xDS - ie the protocol name and is not exactly an acronym

ive updated the PR on that basis, and opened a glossary PR where i think we can pin down these terms.